WcaOnRails/app/views/competitions_mailer/submit_report_nag.html.erb

```diff
@@ -0,0 +1,14 @@
+<p>
+  Dear <%= "Delegate".pluralize(@competition.delegates.count) %> of <%= @competition.name %>,
+</p>
+
+<p>
+  This is an automated reminder. Over a week has passed since <%= @competition.name %> took
+  place and the delegate report of the competition is still not available for The Board
+  as required by regulation <a href="https://www.worldcubeassociation.org/regulations/#1c1">1c1)</a>.
+  Please submit the report as fast as possible <%= link_to "here", delegate_report_url(@competition) %>.
+</p>
+
```

Comment:
Can we add an additional paragraph here? Kinda...

"Irrespective of if you have availability to send in your report any soon, please reply this email to update us on the situation of this missing report."

Something like that, feel free to word it your way.